bpython/test/test_interpreter.py

```diff
@@ -62,7 +55,7 @@ def f():
         def g():
             return f()
 
-        i.runsource('g()', encode=False)
+        i.runsource('g()')
```

Comment:
I've removed encode=False after coming to the conclusion that it should have been True the whole time: line numbers were being reported incorrectly in Python 2 previously, but now they're right. This test was passing before, but shouldn't have been.